### PR TITLE
Data for captivePortal

### PR DESCRIPTION
**Pull request by rebloor**
Includes:
* update to permissions.json to include captivePortal
* new captivePortal.json

See https://bugzilla.mozilla.org/show_bug.cgi?id=1545159

A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any

### CODE REVIEW

**Comment by rebloor** [comment]
Hi @wbamberg looking for a double check on this content, thanks!

File: webextensions/api/captivePortal.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+              "firefox_android": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              }
+            }
+          }
+        },
+        "getLastChecked": {
```
Comment (wbamberg) [inline webextensions/api/captivePortal.json]: It's good practice I think to order features, so this should come before `getState`.

File: webextensions/api/captivePortal.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+              "firefox_android": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              }
+            }
+          }
+        },
+        "reset": {
```
Comment (wbamberg) [inline webextensions/api/captivePortal.json]: Should be `onConnectivityAvailable` I think - and ordered before `onStateChanged`.

Comment (rebloor) [reply webextensions/api/captivePortal.json]: Thanks, done

File: webextensions/api/captivePortal.json (hunk 1 above)
Comment (rebloor) [reply webextensions/api/captivePortal.json]: Thanks, noted and done

**Comment by rebloor** [comment]
Thanks @wbamberg I have addressed those issues, please take another look

**Pull request merged**